examples/gguf-split/gguf-split.cpp

```diff
@@ -1,31 +1,34 @@
 #include "llama.h"
-#include "ggml.h"
 #include "common.h"
 
 #include <algorithm>
 #include <cmath>
-#include <cstdint>
 #include <cstdlib>
 #include <fstream>
-#include <ios>
 #include <string>
 #include <vector>
 
 #include <stdio.h>
-#include <fcntl.h>
 #include <string.h>
+#include <climits>
+#include <stdexcept>
+
+#if defined(_WIN32)
+    #include <windows.h>
+    #ifndef PATH_MAX
+        #define PATH_MAX MAX_PATH
+    #endif
+    #include <io.h>
+#endif
 
 enum split_operation : uint8_t {
     SPLIT_OP_SPLIT,
     SPLIT_OP_MERGE,
 };
 
-static const char * const LLM_KV_GENERAL_SPLIT_I_SPLIT = "general.split";
-static const char * const LLM_KV_GENERAL_SPLIT_N_SPLIT = "general.split_count";
-
-static const int SPLIT_FILENAME_MAX = 256;
-
-static const char * const SPLIT_FILENAME_FORMAT = "%s-%05d-of-%05d.gguf";
+static const char * const LLM_KV_GENERAL_SPLIT_I_SPLIT   = "split.no";
+static const char * const LLM_KV_GENERAL_SPLIT_N_SPLIT   = "split.count";
+static const char * const LLM_KV_GENERAL_SPLIT_N_TENSORS = "split.tensors.count";
```

Comment:
Match the variable names with the ones in `llama.cpp` for consistency

Reply:
@ggerganov Since we are renaming `general.split` to `split.no`, `general.split_count` to `split.count` and adding `split.tensors.count`, the previous split created in #6204 for example will not be possible to be loaded here. master `gguf-split` only will be able to merge them. Then they should be split again with this code base. is it ok ? or should we keep master key names ?